docs/nos-client/api.md

```diff
@@ -5,17 +5,58 @@ sidebar_label: API Overview
 ---
 
 **NOTE: The API is in active development and may change frequently.  Be sure to check back
-regularly. All API functions will have a maximum of ONE argument: a JSON object containing all the necessary info.**
+regularly. All API functions will have a maximum of ONE argument: a JSON object containing all the
+necessary info.**
 
 The nOS client provides a prebuilt API that is still growing.  For any dApp running through the
-client, the API can be accessed via `window.NOS.V1`.  All functions return a
-[`Promise`](https://www.google.com/search?q=js+promise&oq=js+promise&aqs=chrome..69i57j69i60l3.731j0j7&sourceid=chrome&ie=UTF-8), which can be used to determine if the call succeeded or failed.  Some calls will fail if the
-user rejects the action.
+client, the API can be accessed via `window.NOS.V1`.
 
 
+## Events
 
+Some events have been exposed to provide insight pertaining to network and blockchain changes.  The
+`on` and `once` functions permit subscribing to an event indefinitely or once, respectively.  Each
+of these calls returns a subscription identifier, which can be used to unsubscribe via the `off`
+function.
 
-## `getAddress`
+```javascript
+// subscribe indefinitely
+const subscription = nos.on('someEvent', () => { /* callback */ });
+
+// subscribe once (automatically unsubscribes after first event)
+const subscription = nos.once('someEvent', () => { /* callback */ });
+
+// unsubscribe (optional)
+nos.off('block', subscription);
```

Comment:
It would make more sense to do the following: 
```
// subscribe indefinitely
const subscription = nos.on('someEvent', () => { /* callback */ });

 // unsubscribe
nos.off('block', subscription);

 // subscribe once (automatically unsubscribes after first event)
const subscription = nos.once('someEvent', () => { /* callback */ });
```